nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -0,0 +1,114 @@
+import datetime
+
+from django.utils.translation import gettext_noop as _
+from django.utils import timezone
+from django_datastream import datastream
+
+from nodewatcher.core.monitor import processors as monitor_processors
+from nodewatcher.modules.monitor.sources.http import processors as http_processors
+from nodewatcher.modules.monitor.datastream import fields as ds_fields, models as ds_models
+from nodewatcher.modules.monitor.datastream.pool import pool as ds_pool
+
+
+class SurveyInfoStreams(ds_models.RegistryRootStreams):
+    neighbor_graph = ds_fields.GraphField(tags={
+        'title': _("Neighborhood topology"),
+        'description': _("Neighborhood topology."),
+    })
+
+    def get_module_name(self):
+        return 'monitor.http.survey'
+
+    def get_stream_highest_granularity(self):
+        # The nodewatcher-agent performs a survey once every ~240 monitoring intervals according to
+        # https://github.com/wlanslovenija/nodewatcher-agent/blob/master/modules/wireless.c#L362
+        # and a monitoring run is performed every 30 seconds according to
+        # https://github.com/wlanslovenija/nodewatcher-agent/blob/73f2b25db2c34ae7d70904bac31379a2243bade0/modules/wireless.c#L437
+        # (master branch at https://github.com/wlanslovenija/nodewatcher-agent/blob/master/modules/wireless.c#L437).
```

Comment:
A agree. In general I would prefer something like:

> The nodewatcher-agent performs a survey once every ~240 monitoring intervals and a monitoring run is performed every 30 seconds according to the wireless module of nodewatcher-agent.

Because even links to the code change. And implementations as well.

What do you think?